src/main/java/edu/hawaii/its/api/wrapper/FindAttributesResults.java

```diff
@@ -24,6 +24,16 @@ public FindAttributesResults(WsFindAttributeDefNamesResults wsFindAttributeDefNa
         return this.wsFindAttributeDefNamesResults.getResultMetadata().getResultCode();
     }
 
+    public AttributesResult getResult() {
+        WsAttributeDefName[] wsAttributeDefNames = this.wsFindAttributeDefNamesResults.getAttributeDefNameResults();
+        AttributesResult attributesResult = new AttributesResult();
+        if (isEmpty(wsAttributeDefNames)) {
+            return attributesResult;
+        }
+        attributesResult = new AttributesResult(wsAttributeDefNames[0]);
+        return attributesResult;
+    }
+
```

Comment:
Is this function being used?

Reply:
Yes, I just modified the code to use `getResult()`. I wrote that function since the result is being given back in just one index so I thought it would be better to create a new function without the for loop.